acceptance/tests/pxp-module-puppet/run_puppet_nonASCII.rb

```diff
@@ -0,0 +1,58 @@
+require 'pxp-agent/test_helper.rb'
+
+ENVIRONMENT_NAME = 'non_ASCII'
+
+test_name 'C98107 - Run puppet with non-ASCII characters in Puppet code' do
+
+  step 'On master, create a new environment that includes a non-ASCII character and will result in Changed' do
+    environmentpath = master.puppet['environmentpath']
```

Comment:
we have a helper for this, and this environment needs a teardown.  
actually we should probably just use mk_tmp_environment_on() or whatever it's called in our libs